### ISSUE TITLE
System.Diagnostics.Process failures on Windows Nano

### ISSUE THREAD

**Issue by Priya91** [dotnet/corefx#6622]
```
Discovering: System.Diagnostics.Process.Tests
14:11:45 Discovered:  System.Diagnostics.Process.Tests
14:11:45 Starting:    System.Diagnostics.Process.Tests
14:11:46    **System.Diagnostics.Tests.ProcessStandardConsoleTests.TestChangesInConsoleEncoding** [FAIL]
14:11:46       Assert+WrapperXunitException : File path: d:\j\workspace\outerloop_win---ce410c48\src\System.Diagnostics.Process\tests\ProcessStandardConsoleTests.cs. Line: 26
14:11:46       ---- Assert.Equal() Failure
14:11:46       Expected: 65001
14:11:46       Actual:   437
14:11:46       Stack Trace:
14:11:46 TestUserCredentialsPropertiesOnWindows: NetUserAdd failed: The account already exists
14:11:46             at Assert.WrapException(Exception e, String callerFilePath, Int32 callerLineNumber)
14:11:46             at Assert.Equal[T](T expected, T actual, String path, Int32 line)
14:11:46             at System.Diagnostics.Tests.ProcessStandardConsoleTests.<TestChangesInConsoleEncoding>b__1_0(Int32 expectedCodePage)
14:11:46             at System.Diagnostics.Tests.ProcessStandardConsoleTests.TestChangesInConsoleEncoding()
14:11:46          ----- Inner Stack Trace -----
14:11:46             at Assert.Equal[T](T expected, T actual, String path, Int32 line)
14:11:48   ** System.Diagnostics.Tests.ProcessThreadTests.TestStartAddressProperty** [FAIL]
14:11:48       Assert+WrapperXunitException : File path: d:\j\workspace\outerloop_win---ce410c48\src\System.Diagnostics.Process\tests\ProcessThreadTests.cs. Line: 113
14:11:48       ---- Assert.Equal() Failure
14:11:48       Expected: False
14:11:48       Actual:   True
14:11:48       Stack Trace:
14:11:48             at Assert.WrapException(Exception e, String callerFilePath, Int32 callerLineNumber)
14:11:48             at Assert.Equal[T](T expected, T actual, String path, Int32 line)
14:11:48             at System.Diagnostics.Tests.ProcessThreadTests.TestStartAddressProperty()
14:11:48          ----- Inner Stack Trace -----
14:11:48             at Assert.Equal[T](T expected, T actual, String path, Int32 line)
14:11:49    **System.Diagnostics.Tests.ProcessWaitingTests.MultipleProcesses_StartAllKillAllWaitAll **[FAIL]
14:11:49       System.ComponentModel.Win32Exception : Not enough quota is available to process this command
14:11:49       Stack Trace:
14:11:49             at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
14:11:49             at System.Diagnostics.Process.Start()
14:11:49             at System.Diagnostics.Tests.ProcessWaitingTests.MultipleProcesses_StartAllKillAllWaitAll()
```

Test result [here](http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/outerloop_winnano_debug_tst/6/console)

**Comment by Priya91** [comment]
This is fixed by #8016 and #7786

**Comment by Priya91** [comment]
The net user add failed on a very early build of nano with corrupted test user account data.
